### PR TITLE
Add Fleet/Agent release notes for 7.17.2

### PR DESCRIPTION
**Pull request by dedemorton**
Adds release notes for 7.17.2 and fixes commented out template.

### CODE REVIEW

**Review by jen-huang — APPROVED** [review]
Fleet LGTM

**Review by ph — CHANGES_REQUESTED** [review]
We are missing this important commit

Fix the start sequence of beats that was non-deterministic making beats missing their configuration from Agent and not sending events.

https://github.com/elastic/beats/pull/30694

File: docs/en/ingest-management/release-notes/release-notes-7.17.asciidoc [hunk 1]
```diff
@@ -21,6 +23,30 @@ Also see:
...
+
+{fleet}::
+* Use validated fields for `default_fields` index setting. {kib-pull}128094[#128094]
+* Fix links to Agent logs for APM, Endpoint, synthetics, and osquery. {kib-pull}127480[#127480]
+* Make input IDs unique in agent policy yaml. {kib-pull}127343[#127343]
+
+{agent}::
+* Propagate input ID from the Agent policy into the Filebeat configuration. Note
+that no validation is performed on this field. {agent-pull}30386[#30386]
+* Fix the start sequence of beats that was non-deterministic making beats missing their
```
[the remark this answers is not in the document]
Comment (ph) [reply docs/en/ingest-management/release-notes/release-notes-7.17.asciidoc]: 👍🏽

**Comment by ph** [comment]
LGTM, I will check the 8.1.2, if this entry is missing too.

**Pull request merged**